doc/dev/crimson/crimson.rst

```diff
@@ -64,6 +65,82 @@ The last causes pools to be created by default with the ``crimson`` flag.
 Crimson pools are restricted to operations supported by Crimson.
 ``Crimson-osd`` won't instantiate PGs from non-Crimson pools.
 
+Configure Crimson with Bluestore
+================================
+
+As Bluestore is not a Crimson native `object store backend`_,
+deploying Crimson with Bluestore as the back end requires setting
+one of the two following configuration options:
+
+.. note::
+
+   #. These two options, along with ``crimson_alien_op_num_threads``,
+      can't be changed after deployment.
+   #. `vstart.sh`_ sets this options using the ``--crimson-smp`` flag.
```

Comment:
Suggest moving this note to down below the two options.

Reply:
Also suggest adding a note that when threading is enabled, `cores` here means `vcores` or `threads` not physical cores.

Reply:
Thanks for reviewing! 

> Suggest moving this note to down below the two options.

I'm afraid that moving it will make it seem like its relevant for the second option only.